inc/namespace.php

```diff
@@ -60,31 +65,47 @@ function is_development_environment() : bool {
 		// WordPress exclusions.
 		$exclude &&
 		// If any of the environment checks are true, we're in a dev environment.
-		( $hm_dev || $altis_dev || $other_dev || $creds_defined )
+		( $hm_dev || $altis_dev || $other_dev )
 	) {
 		return true;
 	}
 
 	return false;
 }
 
+/**
+ * Enable settings on production.
+ *
+ * @return bool
+ */
+function enable_auth(): bool {
+	if (
+		! is_super_admin() && defined( 'HM_ENV_TYPE' ) && HM_ENV_TYPE === 'production' ) {
```

Comment:
spacing here is weird. either put all the conditions on the same line
```php
if ( ! is_super_admin() && defined( 'HM_ENV_TYPE' ) && HM_ENV_TYPE === 'production' )
```
or break it up on a separate line
```php
if (
    ! is_super_admin() && defined( 'HM_ENV_TYPE' ) && HM_ENV_TYPE === 'production' 
)
```
or break each condition into its own line
```php
if (
    ! is_super_admin() && 
    defined( 'HM_ENV_TYPE' ) && 
    HM_ENV_TYPE === 'production' 
)
```